### ISSUE TITLE
No support for backward integrations

### ISSUE THREAD

**Issue by RoryBarnes** [VirtualPlanetaryLaboratory/vplot#1]
It looks like vplot can only handle .forward files. For the EarthMoonTides example in the vplanet repo, if I rename the file extensions to .forward, vplot works.

**Comment by rodluger** [comment]
Will fix this. Also, vplot seems to choke when there are _no_ forward/backward files. This should be an easy fix. https://travis-ci.com/VirtualPlanetaryLaboratory/vplanet-private/builds/81330272

**Comment by rodluger** [comment]
@RoryBarnes I added support for backward integrations in `vplot`. To enable it, re-install it from source (I'm waiting to update the `pip` version until I've fixed some more issues).

**Comment by rodluger** [comment]
@RoryBarnes I added an example to `vplanet-private` called `backward` to showcase a backward integration, but there's something wrong with the `vplanet` run. What I did was (1) run a simple eqtide circularization simulation forward in time, then (2) take the final `(a, e)` state of the planet and set that as the initial state for a backward simulation, which I ran for the same amount of time. Here's what I get:

![image](https://user-images.githubusercontent.com/9323819/44620607-2bc34b00-a84c-11e8-8cbd-d0c158f21dc0.png)

The sign on `da/dt` and `de/dt` in the backward evolution is wrong! Am I doing something wrong, or is this a bug in `vplanet`? The example lives [here](https://github.com/VirtualPlanetaryLaboratory/vplanet-private/tree/master/examples/backward).
